WcaOnRails/app/helpers/static_pages_helper.rb

```diff
@@ -1,9 +1,8 @@
 module StaticPagesHelper
-  def format_team_members(team)
-    leader_team = :"#{team}_leader"
-    User.where(team => true).order(leader_team => :desc, name: :asc).map do |u|
-      s = u.name
-      if u.send(leader_team)
+  def format_team_members(team_friendly_id)
+    TeamMember.where(team_id: Team.find_by_friendly_id!(team_friendly_id).id).order(:team_leader => :desc).map do |u|
```

Comment:
To order by an association, I think you need to do something like what they recommend on http://stackoverflow.com/a/19617397/1739415:

```ruby
TeamMember.where(team_id: Team.find_by_friendly_id!(team_friendly_id).id).include(:user).order(team_leader: desc, 'user.name asc')
```